### ISSUE TITLE
Support getting the type of a field

### ISSUE THREAD

**Issue by Pajn** [MasterLinux/ClassLoader#1]
It would be very useful if one could get type of a field (type if it is an variable or return type if it is an getter).

To clarify I mean type static type of the field. The runtime type can be acquired by getting the field and checking the runtime type of the object.

**Comment by MasterLinux** [comment]
Yeah, I will implement this functionality in version `0.0.3`. Just for clarifying it on my side. Let's say we have a setter like

``` dart
set isLoading(bool loading) => _isLoading = loading;
```

you will check whether `isLoading` is type of `bool`, right? For example

``` dart
var isBool = classLoader.fields[#isLoading].isTypeOf(#bool);
```
